packages/namespaces-react/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@redux-tools/namespaces-react",
+  "version": "0.8.2",
+  "license": "MIT",
+  "main": "lib/redux-tools-namespaces-react.js",
+  "module": "es/redux-tools-namespaces-react.js",
+  "unpkg": "dist/redux-tools-namespaces-react.js",
+  "description": "React bindings for the @redux-tools/namespaces package.",
+  "repository": "https://github.com/lundegaard/redux-tools",
+  "contributors": [
+    "Tomas Konrady <tomas.konrady@lundegaard.eu>",
+    "Vaclav Jancarik <vaclav.jancarik@lundegaard.eu>",
+    "Martin Kadlec <martin.kadlec@lundegaard.eu>",
+    "Sergey Dunaevskiy <sergey.dunaevskiy@lundegaard.eu>",
+    "Veronika Wernerova <veronika.wernerova@lundegaard.eu>"
+  ],
+  "dependencies": {
+    "@redux-tools/namespaces": "^0.8.2",
+    "@redux-tools/reducers": "^0.8.2",
+    "@redux-tools/utils-react": "^0.8.2",
+    "prop-types": "^15.7.2",
+    "ramda": ">=0.26.1",
+    "ramda-extension": ">=0.8.0"
+  },
+  "peerDependencies": {
+    "react": ">=16.8.3",
+    "react-redux": "^6.0.0 || ^7.0.0",
+    "redux": "4.0.5"
```

Comment:
never use exact versions in peer dependencies! please use "redux": "^4.0.4" like in the rest of the project.